lte/gateway/c/core/BUILD.bazel

```diff
@@ -692,13 +692,15 @@ MME_HDRS = [
     "oai/tasks/mme_app/mme_app_timer.h",
     "oai/tasks/mme_app/mme_app_timer_management.hpp",
     "oai/tasks/mme_app/mme_app_ueip_imsi_map.h",
-    "oai/tasks/nas/api/mme/mme_api.h",
-    "oai/tasks/nas/api/network/nas_message.h",
-    "oai/tasks/nas/emm/EmmCommon.h",
-    "oai/tasks/nas/emm/LowerLayer.h",
-    "oai/tasks/nas/emm/emm_data.h",
-    "oai/tasks/nas/emm/emm_main.h",
-    "oai/tasks/nas/emm/emm_proc.h",
+    "oai/tasks/nas/api/mme/mme_api.hpp",
+    "oai/tasks/nas/api/network/nas_message.hpp",
+    "oai/tasks/nas/emm/EmmCommon.hpp",
+    "oai/tasks/nas/emm/LowerLayer.hpp",
+    "oai/tasks/nas/emm/emm_data.hpp",
+    "oai/tasks/nas/emm/emm_headers.hpp",
+    "oai/tasks/nas/emm/emm_main.hpp",
+    "oai/tasks/nas/emm/emm_proc.hpp",
+    "oai/tasks/nas/emm/emm_headers.hpp",
```

Comment:
`"oai/tasks/nas/emm/emm_headers.hpp",` appears twice. In lines 700 and 703.

Reply:
Resolved